components/query.py

```diff
@@ -109,28 +109,20 @@ def get_images(df, subspecies, view, sex, hybrid, num_images):
     --------
     Imgs - List of html image elements with `src` element pointing to paths for the requested number of images matching given parameters.
            Returns html header4 "No Such Images. Please make another selection." if no images matching parameters exist.
-           Returns html header4 indicating number of matching entries without filename or filepath.
+           Returns html header4 indicating number of matching entries without filepath(s).
     '''
     try:
-        filenames, filepaths = get_filenames(df, subspecies, view, sex, hybrid, num_images)
+        filepaths = get_filenames(df, subspecies, view, sex, hybrid, num_images)
     except ValueError as e:
         return html.H4(str(e) + " Please make another selection.", 
                     style = PRINT_STYLE)
-    Imgs = []
-    for i in range(len(filenames)):
-        if filenames[i] in filepaths[i]:
-            image_path = filepaths[i]
-        else:
-            if filepaths[i][-1] == '/':
-                image_path = filepaths[i] + filenames[i]
-            else:
-                image_path = filepaths[i] + '/' + filenames[i]
-        Imgs.append(html.Img(src = image_path, style = IMG_STYLE))
+    Imgs = [html.Img(src = filepaths[i], style = IMG_STYLE) for i in range(len(filepaths))]
+    
     return Imgs
 
 def get_filenames(df, subspecies, view, sex, hybrid, num_images):
     '''
-    Funtion to randomly select the given number of filenames for images adhering to specified filters.
+    Function to randomly select the given number of filepaths (file urls) for images adhering to specified filters.
```

Comment:
```suggestion
    randomly selects the given number of filepaths (file urls) for images adhering to specified filters.
```

Very minor suggestion. It's a function by definition, so might as well not repeat that fact.

Reply:
This file has many like this, and `dashboard.py` has some too. I'll standardize them all to be consistently more direct like in `graphs.py`.